docs/COMPILATION.MD

```diff
@@ -186,7 +186,45 @@ Once set up you can run the GUI interface from the terminal `./ccextractorGUI`
 
 ## Windows
 
-Open the windows/ccextractor.sln file with Visual Studio (2015 at least), and build it.
+<h1>Compilation guide</h1>
+<h4>*First of all you have to have your visual studio installed on your computer.</h4>
+
+<p>When you already have visual studio you have to download ccextractor code <a href="https://github.com/CCExtractor/ccextractor/">CCExtractor</a></p>
+  
+<p>After downloading open your CCExtractor folder and then open windows folder.</p>
+<br>
+<h4>Here you can see two files .vcxproj(marked with blue arrows) and one .sln project(marked with a red arrow)</h4>
+
+<img src="img/1.png">
+
+<br>
+
+<p>You have to open .sln if you want to build both non GUI and GUI ccextractor or you can build them seperately opening .vcxproj files one by one.</p>
+
+<br>
+
+<h4>Further, if you receive a confirmation request about downloading from unsafe resource just click OK and go forward</h4>
+<br>
+<img src="img/2.png">
+
+<br>
+<h4>Then if you opened .sln file you will have 2 projects on the right pannel GUI and non GUI and you can also open them separetely by opening .vcxproj files.</h4>
```

Comment:
remove h4 and br
for newline you can use 2 space at end of line before new line